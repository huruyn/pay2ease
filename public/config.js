window.API_URL = "https://zippy-retail-api-dev.azurewebsites.net";