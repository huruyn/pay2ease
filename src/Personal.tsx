import { useMsal } from '@azure/msal-react';
import { useTranslation, Trans } from 'react-i18next';
import Circle_Grey_01 from './assets/img/background/Circle_Grey-01.jpg';
import orangered from './assets/img/background/orangered.jpg';
import ZippyCash_Icons_H from './assets/img/roundicons/personal/ZippyCash_Icons_H.png';
import ZippyCash_Icons_I from './assets/img/roundicons/personal/ZippyCash_Icons_I.png';
import ZippyCash_Icons_J from './assets/img/roundicons/personal/ZippyCash_Icons_J.png';
import PersonalBg from './assets/img/background/help_my_per_hero.jpg';
import BlueCircle from './assets/img/roundicons/business/bluecircle.png';
import NumberCircle1 from './assets/img/roundicons/business/numbercircle_1.png';
import NumberCircle2 from './assets/img/roundicons/business/numbercircle_2.png';
import NumberCircle3 from './assets/img/roundicons/business/numbercircle_3.png';
import NumberCircle4 from './assets/img/roundicons/business/numbercircle_4.png';
import { b2cPolicies, loginRequest } from './authConfig';
import Footer from './footer/Footer';

export default function Personal() {
  const { instance } = useMsal();
  const { t } = useTranslation();

  return (
    <>
      <div
        className="bg-light"
        style={{
          backgroundImage: `url(${Circle_Grey_01})`,
          backgroundSize: 'cover',
        }}
      >
        <div className="container">
          <div className="row py-5">
            <div className="col">
              <div className="mb-3">
                <h2 style={{ color: '#FF1926' }}>
                  {t('personal.personal_account')}
                </h2>
                <h1
                  className="redtitle hero-heading display-3"
                  style={{ fontSize: '4em', paddingBottom: '20px' }}
                >
                  {t('personal.header_title')}
                </h1>
                <button
                  type="button"
                  className="btn rounded-0 my-buttonred"
                  onClick={() => instance.loginRedirect({
                    ...loginRequest,
                    authority: b2cPolicies.authorities.signUpPersonal.authority,
                  })}
                >
                  {t('personal.sign_up_btn')}
                </button>
              </div>
            </div>
          </div>
          <img
            src={orangered}
            alt=""
            className="orangeredbar position-absolute bottom-0"
          />
          <div className="row mt-4 justify-content-center">
            <div
              className="col-12 col-md-6 col-lg-4 mb-4 position-relative px-5"
              style={{ zIndex: 1 }}
            >
              <div className="mb-4">
                <img
                  src={ZippyCash_Icons_H}
                  alt={t('personal.image_alt_1')}
                  className="img-fluid centerimage"
                  data-aos="flip-left"
                />
              </div>
              <div>
                <h3 className="redtitle text-center">
                  {t('personal.send_btn')}
                </h3>
                <p className="content-home">{t('personal.send_description')}</p>
              </div>
            </div>
            <div className="col-12 col-md-6 col-lg-4 mb-4 position-relative px-5">
              <div className="mb-4">
                <img
                  src={ZippyCash_Icons_I}
                  alt={t('personal.image_alt_2')}
                  className="img-fluid centerimage"
                  data-aos="flip-left"
                  data-aos-delay="200"
                />
              </div>
              <div>
                <h3 className="bluetitle text-center">
                  {t('personal.spend_btn')}
                </h3>
                <p className="content-home">
                  {t('personal.spend_description')}
                </p>
              </div>
            </div>
            <div className="col-12 col-md-6 col-lg-4 mb-4  position-relative px-5">
              <div className="mb-4">
                <img
                  src={ZippyCash_Icons_J}
                  alt={t('personal.image_alt_3')}
                  className="img-fluid centerimage"
                  data-aos="flip-left"
                  data-aos-delay="400"
                />
              </div>
              <div>
                <h3 className="redtitle text-center">
                  {t('personal.funzpoints')}
                </h3>
                <p className="content-home">
                  {t('personal.funzpoints_description')}
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <main style={{ backgroundColor: 'white' }}>
        <div className="container">
          <div className="row pt-5 pb-4">
            <div className="col-12 col-md-11 col-lg-9">
              <div className="mb-3">
                <h1
                  className="redtitle hero-heading display-3 mb-4"
                  style={{ fontSize: '4em' }}
                >
                  {t('personal.personal_title')}
                </h1>
                <p className="content-home">
                  {t('personal.personal_desc')}
                </p>
              </div>
            </div>
            <div className="col-12 col-md-1 col-lg-3" />
          </div>
          <div className="row">
            <div className="col-12 col-md-5 col-lg-6">
              <h4 className="redtitle">{t('personal.personal_sub_title1')}</h4>
              <ul>
                <li>
                  {t('personal.personal_sub_desc_1')}
                </li>
                <li>{t('personal.personal_sub_desc_2')}</li>
                <li>
                  {t('personal.personal_sub_desc_3')}
                </li>
              </ul>
              <br />
              <h4 className="redtitle">{t('personal.personal_sub_title2')}</h4>
              <ul>
                <li>{t('personal.personal_sub_desc_4')}</li>
                <li>{t('personal.personal_sub_desc_5')}</li>
              </ul>
              <h1 className="redtitle mt-5">{t('personal.signup_free')}</h1>
              <h2 className="redsubtitle mb-5">
                {t('personal.no_need_back')}
              </h2>
            </div>
            <div
              className="col-12 col-md-7 col-lg-6 backgroundimage"
              style={{
                backgroundImage: `url(${PersonalBg})`,
                backgroundSize: 'cover',
              }}
            />
          </div>
        </div>
        <div style={{ backgroundColor: '#51aad8' }}>
          <div className="container">
            <div className="row">
              <div className="col-12 col-md-6 col-lg-3">
                <div
                  className="position-relative top-0"
                  data-aos="flip-left"
                  data-aos-delay="0"
                >
                  <img
                    src={BlueCircle}
                    alt=""
                    className="img-fluid centerimage bluecircle"
                  />
                  <img
                    src={NumberCircle1}
                    alt=""
                    className="position-absolute top-0 start-50 numbercicle"
                  />

                  <div className="position-absolute halfcirclecopy">
                    <h5 className="whitetitle text-center">
                      <a href="business-signup.html" className="whitetitle">
                        <Trans i18nKey="personal.signup_here" />
                      </a>
                    </h5>
                  </div>
                </div>
              </div>
              <div className="col-12 col-md-6 col-lg-3">
                <div
                  className="position-relative top-0"
                  data-aos="flip-left"
                  data-aos-delay="200"
                >
                  <img
                    src={BlueCircle}
                    alt=""
                    className="img-fluid centerimage bluecircle"
                  />
                  <img
                    src={NumberCircle2}
                    alt=""
                    className="position-absolute top-0 start-50 numbercicle"
                  />

                  <div className="position-absolute halfcirclecopy">
                    <h5 className="whitetitle text-center">
                      <Trans i18nKey="personal.send_text_email" />
                    </h5>
                    <p className="whitecopy text-center">
                      <Trans i18nKey="personal.choose_works" />
                    </p>
                  </div>
                </div>
              </div>
              <div className="col-12 col-md-6 col-lg-3">
                <div
                  className="position-relative top-0"
                  data-aos="flip-left"
                  data-aos-delay="400"
                >
                  <img
                    src={BlueCircle}
                    alt=""
                    className="img-fluid centerimage bluecircle"
                  />
                  <img
                    src={NumberCircle3}
                    alt=""
                    className="position-absolute top-0 start-50 numbercicle"
                  />

                  <div className="position-absolute halfcirclecopy">
                    <h5 className="whitetitle text-center">
                      <Trans i18nKey="personal.link_download" />
                    </h5>
                    <p className="whitecopy text-center">
                      <Trans i18nKey="personal.link_download_desc" />
                    </p>
                  </div>
                </div>
              </div>
              <div className="col-12 col-md-6 col-lg-3">
                <div
                  className="position-relative top-0"
                  data-aos="flip-left"
                  data-aos-delay="600"
                >
                  <img
                    src={BlueCircle}
                    alt=""
                    className="img-fluid centerimage bluecircle"
                  />
                  <img
                    src={NumberCircle4}
                    alt=""
                    className="position-absolute top-0 start-50 numbercicle"
                  />

                  <div className="position-absolute halfcirclecopy">
                    <h5 className="whitetitle text-center">
                      <Trans i18nKey="personal.open_and_start" />
                    </h5>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>
      <Footer />
    </>
  );
}
