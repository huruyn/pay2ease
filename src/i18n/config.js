import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

i18n.use(initReactI18next).init({
  fallbackLng: 'en-US',
  lng: 'en-US',
  resources: {
    'en-CA': {
      translations: require('./locales/en-CA/translations.json')
    },
    'en-US': {
      translations: require('./locales/en-US/translations.json')
    },
    'es-US': {
      translations: require('./locales/es-US/translations.json')
    },
    'fr-CA': {
      translations: require('./locales/fr-CA/translations.json')
    },
  },
  ns: ['translations'],
  defaultNS: 'translations'
});

i18n.languages = ['en-CA', 'en-US', 'es-US', 'fr-CA'];

export default i18n;