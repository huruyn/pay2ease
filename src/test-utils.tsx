import { FC, ReactElement } from 'react';
import { render, RenderOptions } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import './i18n/config';
import { store } from './app/store';

const AllTheProviders: FC = ({ children }) => (
  <Provider store={store}>
    <BrowserRouter>{children}</BrowserRouter>
  </Provider>
);

const customRender = (
  ui: ReactElement,
  options?: Omit<RenderOptions, 'wrapper'>,
) => render(ui, { wrapper: AllTheProviders, ...options });

export * from '@testing-library/react';
export { customRender as renderWithProviders };
