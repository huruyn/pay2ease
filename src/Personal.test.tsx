import { renderWithProviders, fireEvent } from './test-utils';
import './i18n/config';
import Personal from './Personal';

const ReactTestRenderer = require('react-test-renderer');

const mockLoginRedirect = jest.fn();
jest.mock('@azure/msal-react', () => ({
  ...(jest.requireActual('@azure/msal-react') as any),
  useMsal: () => ({
    instance: {
      loginRedirect: mockLoginRedirect,
    },
  }),
}));

describe('Personal Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(<Personal />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('click a button on Personal', () => {
    const { container } = renderWithProviders(<Personal />);

    fireEvent.click(container.querySelectorAll('button')[0]);
    expect(mockLoginRedirect).toHaveBeenCalled();
  });
});
