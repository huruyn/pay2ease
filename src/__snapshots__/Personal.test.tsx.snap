// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Personal Component matches the snapshot 1`] = `
Array [
  <div
    className="bg-light"
    style={
      Object {
        "backgroundImage": "url(Circle_Grey-01.jpg)",
        "backgroundSize": "cover",
      }
    }
  >
    <div
      className="container"
    >
      <div
        className="row py-5"
      >
        <div
          className="col"
        >
          <div
            className="mb-3"
          >
            <h2
              style={
                Object {
                  "color": "#FF1926",
                }
              }
            >
              personal account
            </h2>
            <h1
              className="redtitle hero-heading display-3"
              style={
                Object {
                  "fontSize": "4em",
                  "paddingBottom": "20px",
                }
              }
            >
              You can do that with Zippy!
            </h1>
            <button
              className="btn rounded-0 my-buttonred"
              onClick={[Function]}
              type="button"
            >
              Sign up for a personal account
            </button>
          </div>
        </div>
      </div>
      <img
        alt=""
        className="orangeredbar position-absolute bottom-0"
        src="orangered.jpg"
      />
      <div
        className="row mt-4 justify-content-center"
      >
        <div
          className="col-12 col-md-6 col-lg-4 mb-4 position-relative px-5"
          style={
            Object {
              "zIndex": 1,
            }
          }
        >
          <div
            className="mb-4"
          >
            <img
              alt="young man happily looking at his phone while holding a zippy card"
              className="img-fluid centerimage"
              data-aos="flip-left"
              src="ZippyCash_Icons_H.png"
            />
          </div>
          <div>
            <h3
              className="redtitle text-center"
            >
              SEND
            </h3>
            <p
              className="content-home"
            >
              We bet you're wondering, can I send money across the border inexpensively… yes you can! Can I link Zippy to my debit and credit so my transactions are all in one place?…yes you can! Can I Zipp it to my friend Erica for those late night study eats she ordered in? YES YOU CAN. Just say Zipp it to me and it's done!
            </p>
          </div>
        </div>
        <div
          className="col-12 col-md-6 col-lg-4 mb-4 position-relative px-5"
        >
          <div
            className="mb-4"
          >
            <img
              alt="a zippy card tapping a debt machine"
              className="img-fluid centerimage"
              data-aos="flip-left"
              data-aos-delay="200"
              src="ZippyCash_Icons_I.png"
            />
          </div>
          <div>
            <h3
              className="bluetitle text-center"
            >
              SPEND
            </h3>
            <p
              className="content-home"
            >
              We know what you’re thinking. You want to shop cross border and get the best rates. You want to see all your bills in one spot, at one time. You want the lowest fees when you spend using Zippy Cash. We say.: yes yes and yes again. It's the power of positive thinking! (and a hub platform that centralizes best-inclass financial systems and optimizes options based on your specific needs within seconds - but hey, that's just how we Zipp)
            </p>
          </div>
        </div>
        <div
          className="col-12 col-md-6 col-lg-4 mb-4  position-relative px-5"
        >
          <div
            className="mb-4"
          >
            <img
              alt="young woman playfully showing her zippy card"
              className="img-fluid centerimage"
              data-aos="flip-left"
              data-aos-delay="400"
              src="ZippyCash_Icons_J.png"
            />
          </div>
          <div>
            <h3
              className="redtitle text-center"
            >
              FUNZPOINTS
            </h3>
            <p
              className="content-home"
            >
              We like giving you more than you came for. Funzpoints is our social gaming platform that rewards you with entries into daily cash jackpots. Play for free and earn actual dollars you can transfer to your real world wallet! It's so much fun to say "here, have extra!"
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <main
    style={
      Object {
        "backgroundColor": "white",
      }
    }
  >
    <div
      className="container"
    >
      <div
        className="row pt-5 pb-4"
      >
        <div
          className="col-12 col-md-11 col-lg-9"
        >
          <div
            className="mb-3"
          >
            <h1
              className="redtitle hero-heading display-3 mb-4"
              style={
                Object {
                  "fontSize": "4em",
                }
              }
            >
              How does Zippy work?
            </h1>
            <p
              className="content-home"
            >
              There is no fee to sign up for Zippy. We will never surprise you with hidden fees or rates. We will always give you the lowest rate and lowest fee - and we will give you money back for zipping with us through funzpoints, referral rewards and cash back for every transaction you send through Zippy.
            </p>
          </div>
        </div>
        <div
          className="col-12 col-md-1 col-lg-3"
        />
      </div>
      <div
        className="row"
      >
        <div
          className="col-12 col-md-5 col-lg-6"
        >
          <h4
            className="redtitle"
          >
            Request and send money (for free!)
          </h4>
          <ul>
            <li>
              Login to your account, tap your contact and send them money or a request for money.
            </li>
            <li>
              Easy, fast, free and is sent within seconds
            </li>
            <li>
              Send across borders with the best rates, lowest fees and super Zippy speed (in real time!)
            </li>
          </ul>
          <br />
          <h4
            className="redtitle"
          >
            Pay bills (for free!)
          </h4>
          <ul>
            <li>
              Login to your account and add a bill.
            </li>
            <li>
              Pay bills instantly, anytime, anywhere, with no fee!
            </li>
          </ul>
          <h1
            className="redtitle mt-5"
          >
            Sign up free!
          </h1>
          <h2
            className="redsubtitle mb-5"
          >
            (no back account needed)
          </h2>
        </div>
        <div
          className="col-12 col-md-7 col-lg-6 backgroundimage"
          style={
            Object {
              "backgroundImage": "url(help_my_per_hero.jpg)",
              "backgroundSize": "cover",
            }
          }
        />
      </div>
    </div>
    <div
      style={
        Object {
          "backgroundColor": "#51aad8",
        }
      }
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-12 col-md-6 col-lg-3"
          >
            <div
              className="position-relative top-0"
              data-aos="flip-left"
              data-aos-delay="0"
            >
              <img
                alt=""
                className="img-fluid centerimage bluecircle"
                src="bluecircle.png"
              />
              <img
                alt=""
                className="position-absolute top-0 start-50 numbercicle"
                src="numbercircle_1.png"
              />
              <div
                className="position-absolute halfcirclecopy"
              >
                <h5
                  className="whitetitle text-center"
                >
                  <a
                    className="whitetitle"
                    href="business-signup.html"
                  >
                    Sign up
                    <br />
                    here
                  </a>
                </h5>
              </div>
            </div>
          </div>
          <div
            className="col-12 col-md-6 col-lg-3"
          >
            <div
              className="position-relative top-0"
              data-aos="flip-left"
              data-aos-delay="200"
            >
              <img
                alt=""
                className="img-fluid centerimage bluecircle"
                src="bluecircle.png"
              />
              <img
                alt=""
                className="position-absolute top-0 start-50 numbercicle"
                src="numbercircle_2.png"
              />
              <div
                className="position-absolute halfcirclecopy"
              >
                <h5
                  className="whitetitle text-center"
                >
                  We will send you a
                  <br />
                   link via text or email
                </h5>
                <p
                  className="whitecopy text-center"
                >
                  (you choose which
                  <br />
                   works for you)
                </p>
              </div>
            </div>
          </div>
          <div
            className="col-12 col-md-6 col-lg-3"
          >
            <div
              className="position-relative top-0"
              data-aos="flip-left"
              data-aos-delay="400"
            >
              <img
                alt=""
                className="img-fluid centerimage bluecircle"
                src="bluecircle.png"
              />
              <img
                alt=""
                className="position-absolute top-0 start-50 numbercicle"
                src="numbercircle_3.png"
              />
              <div
                className="position-absolute halfcirclecopy"
              >
                <h5
                  className="whitetitle text-center"
                >
                  Click the link to
                  <br />
                   download the app
                </h5>
                <p
                  className="whitecopy text-center"
                >
                  Zippy is available on over
                  <br />
                   400 smart devices
                </p>
              </div>
            </div>
          </div>
          <div
            className="col-12 col-md-6 col-lg-3"
          >
            <div
              className="position-relative top-0"
              data-aos="flip-left"
              data-aos-delay="600"
            >
              <img
                alt=""
                className="img-fluid centerimage bluecircle"
                src="bluecircle.png"
              />
              <img
                alt=""
                className="position-absolute top-0 start-50 numbercicle"
                src="numbercircle_4.png"
              />
              <div
                className="position-absolute halfcirclecopy"
              >
                <h5
                  className="whitetitle text-center"
                >
                  Open the app
                  <br />
                   and start 
                  <br />
                   Zipping!
                </h5>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>,
  <div
    className="footer-content-wrapper"
  >
    <div
      className="quick-links container"
    >
      <div
        className="row"
      >
        <h3
          className="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-2"
        >
          <ul>
            <li>
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                className="text-decoration-none"
                href="/legal"
                onClick={[Function]}
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          className="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      className="container mb-3"
    >
      <div
        className="d-none d-md-block"
      >
        <div
          className="d-flex justify-content-between border-bottom"
        />
        <div
          className="d-flex justify-content-between border-top"
        >
          <div
            className=""
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            className="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        className="d-block d-md-none"
      >
        <div
          className="d-flex border-bottom"
        >
          <div
            className="d-flex flex-column me-2"
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          className="d-flex justify-content-end border-top"
        >
          <div
            className="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>,
]
`;
